### PR TITLE
add version command to yahcli which perform the getVerionInfo query

### LINKED ISSUE #2518: Update yahcli to support the `getVersionInfo` query
### Problem

There is no way to use yahcli to check the deployed version of a network.

### Solution

Add a `version` subcommand to yahcli whose delegate `HapiApiSuite` runs a `getVersionInfo()` query and prints the results to `STDOUT`.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by anighanta**
**Description**:

**Related issue(s)**:

Fixes #2518 

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: test-clients/src/main/java/com/hedera/services/yahcli/commands/system/VersionInfoCommand.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.hedera.services.yahcli.commands.system;
+
```
Comment (qnswirlds) [inline test-clients/src/main/java/com/hedera/services/yahcli/commands/system/VersionInfoCommand.java:2]: missing license header

File: test-clients/src/main/java/com/hedera/services/yahcli/commands/system/VersionInfoCommand.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import java.util.concurrent.Callable;
+
+import static com.hedera.services.yahcli.config.ConfigUtils.configFrom;
+
+@CommandLine.Command(
+		name = "version",
+		subcommands = { picocli.CommandLine.HelpCommand.class },
+		description = "Get the deployed version of a network")
+public class VersionInfoCommand implements Callable<Integer> {
```
Comment (qnswirlds) [inline test-clients/src/main/java/com/hedera/services/yahcli/commands/system/VersionInfoCommand.java:15]: nit: make this class `final`

**Review by qnswirlds — DISMISSED** [review]

**Review by tinker-michaelj — COMMENTED** [review]
We might add a `.withYahcliLogging()` option to the `GetVersionInfo` verb so we can put the `VersionInfoSpec` in the "less-noisy" list [here](https://github.com/hashgraph/hedera-services/blob/master/test-clients/src/main/java/com/hedera/services/yahcli/Yahcli.java#L136); and then when yahcli logging is enabled, use [`COMMON_MESSAGES.info()`](https://github.com/hashgraph/hedera-services/blob/master/test-clients/src/main/java/com/hedera/services/yahcli/output/CommonMessages.java#L39) to print to stdout.

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM

**Pull request merged**